### PR TITLE
Tags for uncategorized tickets

### PR DESCRIPTION
**Pull request by konstantinos-perperidis**
Show which criteria do match to the issues which cannot be categorized into any of the defined ones.

### CODE REVIEW

File: api/jiraApiClient.js [hunk 1]
```diff
@@ -77,6 +77,41 @@ const fetchIssueLinksFromStoriesByRelease = async (
   return result;
 };
 
+const getAllFields = async (axiosInstance,
```
Comment (sgkiokas) [inline api/jiraApiClient.js:80]: Small comment, but maybe some new lines would make those 2 methods more readable.

File: jrr/jrrMain.js [hunk 2]
```diff
@@ -52,7 +53,12 @@ const jrrMain = async (jrrConfig) => {
   const missingOnes = createKeyLinkPairForIssues(difference, jrrConfig.jira.jiraBaseURL);
   shrinkedData.missing = missingOnes;
 
-  logout(axiosInstance, authHeaders);
+  if (!_.isEmpty(missingOnes)) {
```
Comment (sgkiokas) [inline jrr/jrrMain.js:56]: Can the `missingOnes` be renamed to something is more obvious?

File: jrr/utils/util.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+const _ = require('lodash');
+const api = require('../../api/jiraApiClient');
+
+const getUniqueCriteria = (fromObject) => {
+  const allCriteria = _.map(fromObject, (item) => item.criteria && Object.keys(item.criteria));
+  const filteredCriteria = _.filter(allCriteria, (item) => !_.isNil(item));
+  return new Set(_.flattenDeep(filteredCriteria));
+};
+
+const metaFieldsPredicateClosure = (metaFieldsFromCriteria) => function (jiraField) {
```
Comment (sgkiokas) [inline jrr/utils/util.js:11]: Purely for naming convention reasons, can we turn that to an arrow function? Unless the `metaFieldsPredicateClosure.name` is intended to be used for debugging purposes.

**Review by sgkiokas — COMMENTED** [review]
Only a few comments

File: jrr/utils/util.js (hunk 3 above)
Comment (konstantinos-perperidis) [reply jrr/utils/util.js:11]: I think in that particular case its more readable than using an arrow func.

**Review by sgkiokas — APPROVED** [review]
LG

**Pull request merged**